src/libraries/PositionConfig.sol

```diff
@@ -3,17 +3,61 @@ pragma solidity ^0.8.24;
 
 import {PoolKey} from "@uniswap/v4-core/src/types/PoolKey.sol";
 
-// A PositionConfig is the input for creating and modifying a Position in core, set per tokenId
+// A PositionConfig is the input for creating and modifying a Position in core, whos truncated hash is set per tokenId
 struct PositionConfig {
     PoolKey poolKey;
     int24 tickLower;
     int24 tickUpper;
 }
 
-/// @notice Library for computing the configId given a PositionConfig
+/// @notice Library to get and set the PositionConfigId and subscriber status for a given tokenId
 library PositionConfigLibrary {
```

Comment:
needs a comment explaining the setup - that the msb is the 0/1 for subscriber status, and the other 255 bits are the most significant 255 bits of the hash